### PR TITLE
Compability/scheema generation

### PR DESCRIPTION
**Pull request by Dimitreee**
**This pull request includes:**

1. Base scheema defenitions;
2. Simple `json to scheema sdl` parser;
3. Jest & next.js babel.config.js merge;

### CODE REVIEW

File: apps/_compability_app/index.js [hunk 1]
```diff
@@ -20,20 +21,23 @@ const keystone = new Keystone({
...
         } catch (e) {
             console.warn('Keystone.onConnect() Error:', e)
         }
     },
 })
 
-registerSchemas(keystone, [
-    require('./schema/User'),
-])
+const modules_list = fs.readdirSync('./schema').map(file => require(`./schema/${file}`));
```
Comment (pahaz) [inline apps/_compability_app/index.js:37]: why so?

File: apps/_compability_app/index.js [hunk 2]
```diff
@@ -20,20 +21,23 @@ const keystone = new Keystone({
...
+registerSchemas(keystone, modules_list)
 
-const authStrategy = keystone.createAuthStrategy({
+const auth_strategy = keystone.createAuthStrategy({
```
Comment (pahaz) [inline apps/_compability_app/index.js:40]: python like style detected %)

File: apps/_compability_app/schema/Answer.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+const { GQLListSchema } = require('@core/keystone/schema')
+const { Text } = require('@keystonejs/fields')
+const access = require('@core/keystone/access')
+
+const Answer = new GQLListSchema("Answer", {
+    fields: {
+        name: {
```
Comment (pahaz) [inline apps/_compability_app/schema/Answer.js:7]: question, user, answer ?

File: apps/_compability_app/scripts/restriction_task.spec.js [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+
+        fs.writeFileSync(`${tmp_dir}/mock.json`, JSON.stringify(mock_data))
+        restructData(`../${tmp_dir}/mock.json`, "", `${tmp_dir}/result.js`)
+        const restructed_data = require(`../${tmp_dir}/result.js`)
+
+        expect(restructed_data).toStrictEqual(expected)
+    })
+})
+
+let c;
```
[the remark this answers is not in the document]
Comment (Dimitreee) [reply apps/_compability_app/scripts/restriction_task.spec.js:154]: Whops, garbage detected

File: apps/_compability_app/schema/User.js [hunk 5]
```diff
@@ -1,9 +1,20 @@
...
+    ForgotPasswordAction,
+    ForgotPasswordService,
+    RegisterService,
+} = require('@core/keystone/schemas/User')
+const { Relationship } = require('@keystonejs/fields')
 
 const User = BaseUser._override({
     fields: {
         aboutMyself: { type: Wysiwyg },
+        tests: {
```
Comment (pahaz) [inline apps/_compability_app/schema/User.js:13]: No need to connect tests to users. Just results

Comment (Dimitreee) [reply apps/_compability_app/schema/User.js]: Fixed; Results relation will be added later.

File: apps/_compability_app/scripts/restriction_task.spec.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (Dimitreee) [reply apps/_compability_app/scripts/restriction_task.spec.js]: fixed

**Pull request merged**